libraries/main.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+#include <fstream>
+#include "./libraries.h"
+
+
+using  std::string;
+
+namespace config_reader {
+
+    string *read_config(string config_file_path) {
+
+
+        std::ifstream file(PATH);
+        file.open(PATH);
+
+        if (file.is_open()) {
```

Comment:
You should read the file to count the number of lines first but as a proof of concept this works for me